### PR TITLE
Feature: Fetch home data

### PR DESCRIPTION
**Pull request by at-sonvu**

### CODE REVIEW

**Comment by at-daiho** [comment]
:+1: There are no linter violations.

File: MyApp/ViewModel/ListMenu/ListMenuViewModel.swift [hunk 1]
```diff
@@ -69,30 +71,38 @@ final class ListMenuViewModel: MVVM.ViewModel {
...
+        self.menuItems = Array(menuItems)
+    }
+
+    func getCategory() {
+        guard let category = Category.query(categoryID) else { return }
+        self.category = category
+    }
+
+    func getMenuWithCategory() {
+        guard let menuItems = MenuItem.query() else {
```
Comment (at-tuanmai2) [inline MyApp/ViewModel/ListMenu/ListMenuViewModel.swift]: allMenuItems?

File: MyApp/ViewModel/ListMenu/ListMenuViewModel.swift [hunk 2]
```diff
@@ -14,50 +14,52 @@ import RealmS
...
-        }
+    private func setupData() {
         switch type {
         case .Food:
-            return menus[0].menuItems.count
+            title = "Đồ Ăn"
+            getMenuFoodDrink()
         case .Drink:
-            return menus[1].menuItems.count
+            title = "Đồ Uống"
```
Comment (at-tuanmai2) [inline MyApp/ViewModel/ListMenu/ListMenuViewModel.swift]: chắc phải define ra

File: MyApp/ViewModel/PageViewModel/HomePageViewModel.swift [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+        index -= 1
+        return HomeHeaderViewModel(index, menuItem: menuItems[index])
+    }
+
+    func viewModelForFirstPage() -> HomeHeaderViewModel? {
+        guard let menuItems = menuItems else { fatalError("Please call `fetch first`") }
+        return HomeHeaderViewModel(0, menuItem: menuItems[0])
+    }
+
+//    @objc func pageMoveForward(_ pageViewController: UIPageViewController) {
```
Comment (at-tuanmai2) [inline MyApp/ViewModel/PageViewModel/HomePageViewModel.swift]: remove

File: MyApp/ViewModel/PageViewModel/OverviewPageViewModel.swift [hunk 4]
```diff
@@ -42,7 +46,23 @@ final class OverviewPageViewModel: MVVM.ViewModel {
         let vc = OverviewHeaderController()
         guard let index = id else { return }
         vc.index = index + 1
-        pageViewController.setViewControllers([vc], direction: .forward, animated: true, completion: nil)
         pageViewController.delegate?.pageViewController?(pageViewController, didFinishAnimating: true, previousViewControllers: [vc], transitionCompleted: true)
     }
+
+    func getURLImage() -> [String] {
+        guard let menuItems = menuItems else { fatalError("Please call `fetch()`first") }
+        var menusURL: [String] = []
```
Comment (at-tuanmai2) [inline MyApp/ViewModel/PageViewModel/OverviewPageViewModel.swift]: menusURL?

**Pull request merged**